app/controllers/published_pages_controller.rb

```diff
@@ -15,6 +15,18 @@ def show
     guardian.ensure_can_see!(pp.topic)
     @topic = pp.topic
     @canonical_url = @topic.url
+
+    @body_classes = [
```

Comment:
If you make this a `Set.new` instead it will avoid duplicates which is nice to do defenstively.